doc/changes/changes-4.10.xml

```diff
@@ -0,0 +1,312 @@
+</Item>
+
+<!-- 
+ -->
+<Mark>Change definition of <C>IsPGroup</C> to <E>not</E> require finiteness</Mark><Item>
+
+This is a small change in terms of amount of code changed, but we list it here
+as it has a potential (albeit rather unlikely) impact on the code written by &GAP;
+users:
+In the past, the GAP manual entry for <C>IsPGroup</C> defined p-groups as being
+finite groups, which differs from the most commonly used definition for
+p-groups. Note however that there was not actual implication installed from
+<C>IsPGroup</C> to <C>IsFinite</C>, so it always was possible to actually created
+infinite groups in the filter <C>IsPGroup</C>.
+
+In GAP 4.10, we adjusted (in <URL><LinkText>&Hash;1545</LinkText><Link>https://github.com/gap-system/gap/pull/1545</Link></URL>)
+the documentation for <C>IsPGroup</C> to the commonly accepted definition for
+p-groups. In addition, code in the GAP library and in packages using
+<C>IsPGroup</C> was audited and (in a very few cases) adjusted to explicitly check
+<C>IsFinite</C> (see e.g. <URL><LinkText>&Hash;2866</LinkText><Link>https://github.com/gap-system/gap/pull/2866</Link></URL>).
+</Item>
+
+<!-- 
+ -->
+<Mark>Experimental support for using the Julia garbage collector</Mark><Item>
+
+It is now possible to use the garbage collector of the <URL Text="Julia language">https://julialang.org</URL>
+instead of GAP's traditional GASMAN garbage collector. This is partly
+motivated by a desire to allow tight integration with GAP and Julia in the
+future. Warning: right now, this is <E>slower</E>, and also requires a patched
+version of Julia.
+<P/>
+
+Relevant pull requests:
+<URL><LinkText>&Hash;2092</LinkText><Link>https://github.com/gap-system/gap/pull/2092</Link></URL>,
+<URL><LinkText>&Hash;2408</LinkText><Link>https://github.com/gap-system/gap/pull/2408</Link></URL>,
+<URL><LinkText>&Hash;2461</LinkText><Link>https://github.com/gap-system/gap/pull/2461</Link></URL>,
+<URL><LinkText>&Hash;2485</LinkText><Link>https://github.com/gap-system/gap/pull/2485</Link></URL>,
+<URL><LinkText>&Hash;2495</LinkText><Link>https://github.com/gap-system/gap/pull/2495</Link></URL>,
+<URL><LinkText>&Hash;2672</LinkText><Link>https://github.com/gap-system/gap/pull/2672</Link></URL>,
+<URL><LinkText>&Hash;2688</LinkText><Link>https://github.com/gap-system/gap/pull/2688</Link></URL>,
+<URL><LinkText>&Hash;2793</LinkText><Link>https://github.com/gap-system/gap/pull/2793</Link></URL>,
+<URL><LinkText>&Hash;2904</LinkText><Link>https://github.com/gap-system/gap/pull/2904</Link></URL>,
+<URL><LinkText>&Hash;2905</LinkText><Link>https://github.com/gap-system/gap/pull/2905</Link></URL>,
+<URL><LinkText>&Hash;2931</LinkText><Link>https://github.com/gap-system/gap/pull/2931</Link></URL>.
+</Item>
+
+<!-- 
+ -->
+<Mark>libGAP (WIP)</Mark><Item>
+
+We now provide a semi-official way to allow 3rd party code to link GAP as a
```

Comment:
what is "semi-official"? There is no official definition of its meaning. Maybe say "experimental"?